pyproject.toml

```diff
@@ -65,6 +51,16 @@ check_done = "check_done.command:main"
 "Changelog" = "https://github.com/siisurit/check_done/blob/main/CHANGELOG.md"
 "Issue Tracker" = "https://github.com/siisurit/check_done/issues"
 
+[tool.poetry.group.dev.dependencies]
+cryptography = "^44.0.0"
```

Comment:
Don't we need this in the public dependencies? For JTW?

Reply:
Because it's called directly only in the test folder, **BUT** PyJWT calls it indirectly in the main folder, so therefore it should be actually public. This is my bad, I will fix the requirements and make a commit to amend this.